src/components/MainComponent.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Title from './TitleComponent';
+import { TITLES } from '../shared/titles';
+import { FEATURES } from '../shared/features';
+
+class Main extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            title: TITLES,
```

Comment:
state implies that the data will change from user interaction, but I think the title / feature text on the page is just something that we could pass via props.

Lets just pass the data in as props as opposed to keeping it in state.

Reply:
you mean using dot notation and importing into the component rather than passing state? just clarifing